Snabble/UI/Scanner/ShoppingListCellView.swift

```diff
@@ -0,0 +1,175 @@
+            spinner.widthAnchor.constraint(equalToConstant: 28),
+            spinner.heightAnchor.constraint(equalToConstant: 28),
+            spinner.centerYAnchor.constraint(equalTo: imageViewLayuotGuide.centerYAnchor),
+
+            tickLayoutGuide.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -14),
+            tickLayoutGuide.widthAnchor.constraint(equalToConstant: 46),
+            tickLayoutGuide.topAnchor.constraint(equalTo: topAnchor),
+            tickLayoutGuide.bottomAnchor.constraint(equalTo: bottomAnchor),
+
+            nameLabel.leadingAnchor.constraint(equalTo: imageViewLayuotGuide.trailingAnchor, constant: 10),
+            nameLabel.trailingAnchor.constraint(equalTo: tickLayoutGuide.leadingAnchor),
+            nameLabel.topAnchor.constraint(equalTo: topAnchor),
+            nameLabel.bottomAnchor.constraint(equalTo: bottomAnchor),
+
+            checkContainer.trailingAnchor.constraint(equalTo: tickLayoutGuide.trailingAnchor).usingPriority(.defaultHigh),
+            checkContainer.widthAnchor.constraint(equalToConstant: 24).usingPriority(.defaultHigh + 1),
+            checkContainer.heightAnchor.constraint(equalToConstant: 24),
+            checkContainer.centerYAnchor.constraint(equalTo: tickLayoutGuide.centerYAnchor),
+
+            checkImage.leadingAnchor.constraint(equalTo: checkContainer.leadingAnchor, constant: 6),
+            checkImage.trailingAnchor.constraint(equalTo: checkContainer.trailingAnchor, constant: -6),
+            checkImage.topAnchor.constraint(equalTo: checkContainer.topAnchor, constant: 6),
+            checkImage.bottomAnchor.constraint(equalTo: checkContainer.bottomAnchor, constant: -6),
+
+            quantityLabel.leadingAnchor.constraint(equalTo: tickLayoutGuide.leadingAnchor),
+            quantityLabel.trailingAnchor.constraint(equalTo: checkContainer.leadingAnchor),
+            quantityLabel.heightAnchor.constraint(equalToConstant: 24),
+            quantityLabel.centerYAnchor.constraint(equalTo: tickLayoutGuide.centerYAnchor)
+        ])
+    }
+
+    public func prepareForReuse() {
+        self.productImage?.image = nil
+        self.spinner?.isHidden = true
+        self.quantityLabel?.text = nil
+        self.nameLabel?.text = nil
+    }
+
+    public func configure(with nameLabel: NSAttributedString, _ isItemChecked: Bool, _ quantity: String?) {
+        self.quantityLabel?.text = quantity
+        self.checkImage?.isHidden = !isItemChecked
+
+        let alpha = isItemChecked ? 0.3 : 1
+        self.nameLabel?.alpha = alpha
+        self.checkContainer?.alpha = alpha
+        self.quantityLabel?.alpha = alpha
+        self.nameLabel?.attributedText = nameLabel
+    }
+
+    public func configureProductImage(with image: UIImage) {
+        self.productImage?.image = image
+    }
+
+    public func spinnerStartAnimation() {
+        self.spinner?.startAnimating()
+    }
+
+    public func spinnerStopAnimation() {
+        self.spinner?.stopAnimating()
+    }
```

Comment:
If the `spinner` is not private we don't need these methods.
If you want to make the `spinner` private it's fine to use these methods, but I would rename them to `startAnimating()` and `stopAnimating()` to match the activityIndicator interface.